### ISSUE TITLE
Disable workspace grid popup

### ISSUE THREAD

**Issue by cjkr** [mzur/gnome-shell-wsmatrix#225]
Is there any way to completely disable the workspace grid popup such that switching through workspaces does not raise it?

I have unsuccessfully tried using extensions that disable the workspace switcher popup.

**Comment by ebeem** [comment]
Not in preferences, we will need to modify the code base to have such option.
Maybe disable it if `time to show the popup` is `-1`?

I think you just need to add a condition here, in your case probably if you just add a return, it will work
try it out and let me know how it goes, maybe you can open a PR to perform this return if the popup timeout is et to `-1`
```js
 _showWorkspaceSwitcherPopup(toggle) { 
     if (!toggle) {   // && this.settings.get_int('popup-timeout') < 0
         return; 
     } 

     if (Main.overview.visible) { 
         return; 
     } 
  
     if (toggle) { 
         this._addWorkspaceOverviewKeybindings(); 
     } 
```
https://github.com/mzur/gnome-shell-wsmatrix/blob/3748703d33756e74afef6a96a90d625c6120e9f7/wsmatrix%40martin.zurowietz.de/workspacePopup/workspaceManagerOverride.js#L423-L430

**Comment by mzur** [comment]
Anyone please feel free to open a PR for this.

**Comment by peetersdiet** [comment]
> Not in preferences, we will need to modify the code base to have such option. Maybe disable it if `time to show the popup` is `-1`?

I wondered, why -1 and not simply at 0 ms? To me 0 ms of visibility would mean the same as 'don't open'.
And more practically, the current preferences input of this plugin does support a 0, but not negative numbers.
Later I browsed through the releases and saw that there is mention of the 0ms enabling the popup to stay visible as long as `ctrl`+`alt` is held.

So to avoid clashing with that somewhat hidden feature, I added the option to disable the popup from the settings.
I've attached #241 with my changes.
